.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -42,16 +43,14 @@ Examples:
 ### Key points relevant for the review:
 <!--- Describe here what is the most important part that reviewers should focus on. --->
 
+### Tests:
+<!--- Describe the test coverage of the introduced change.
 
-
-<!--- 
-At this point you can hit "Create".
-The checklist below shall be filled in the created PR.
+If you select `Existing tests apply` option, please list which test cases cover the introduced
+functionality. For example:
+- test_operator_gaussian_blur.py: test_gaussian*
+- tensor_vector_test.cc: TensorVectorVariableBatchSizeTest*
```

Comment:
I think it's too detailed - especially with GTest, there can be very many tests and listing them all will just flood the PR message.